frontend/src/employee-frontend/components/vasu/sections/DynamicSections.tsx

```diff
@@ -283,11 +287,21 @@ export function DynamicSections({
   return <>{content}</>
 }
 
-const SectionContent = styled(ContentArea)<{ padBottom: boolean }>`
-  /* make selector specific enough */
-  && {
-    padding-bottom: ${(p) => (p.padBottom ? defaultMargins.L : '0px')};
-  }
+export const SectionContent = styled(ContentArea)<{
+  highlighted: boolean
+  padBottom: boolean
+}>`
+  border-left: 5px solid
+    ${({ highlighted, theme }) =>
+      highlighted ? theme.colors.main.medium : 'transparent'};
+  padding: ${defaultMargins.L};
+  padding-left: calc(${defaultMargins.L} - 5px);
```

Comment:
Should the 5px be removed from here even if it is not highlighted?

Reply:
The border is left there even if the section is not highlighted, it's just transparent.